lib/data/external/GcpClient.js

```diff
@@ -0,0 +1,12 @@
+const { GCP } = require('./GCP');
```

Comment:
why isn't this file in the GCP subdirectory?

Reply:
In the GCP subdirectory, I have an index.js for importing any of the GCP loose files.

Reply:
why have loose files? i meant why is lib/data/external/GcpClient.js not lib/data/external/GCP/GcpClient.js

Reply:
I see. I have the files in the `GCP` subdirectory just for implementing the Google cloud storage APIs. GcpClient, like AwsClient and AzureClient, is a file for implementing the S3 Google backend.